homeassistant/components/prusalink/config_flow.py

```diff
@@ -34,13 +34,38 @@
 )
 
 
+def ensure_printer_is_supported(version: VersionInfo) -> None:
+    """Raise NotSupported exception if the printer is not supported."""
+
+    try:
+        if AwesomeVersion("2.0.0") <= AwesomeVersion(version["api"]):
+            return
+
+        # Workaround to allow PrusaLink 0.7.2 on MK3 and MK2.5 that supports
+        # the 2.0.0 API, but doesn't advertise it yet
+        if (
+            "original" in version
+            and (
+                version["original"].startswith("PrusaLink I3MK3")
+                or version["original"].startswith("PrusaLink I3MK2")
+            )
```

Comment:
```suggestion
            version.get("original", "").startswith(("PrusaLink I3MK3","PrusaLink I3MK2"))
```

Maybe even better would be a regex as only the last digit is different